### ISSUE TITLE
Tutorial documentation still empty.

### ISSUE THREAD

**Issue by jadenyjw** [tflearn/tflearn#498]
http://tflearn.org/tutorials/ section of the documents seems very empty. We should implement generic implementation tutorials for newcomers that would like to create their own image classifier, text classifier, etc.

**Comment by aymericdamien** [comment]
Yes, I agree, it is important to have more tutorials! We got some delayed. If anyone is interested, you are more than welcome to help! :)

**Comment by jadenyjw** [comment]
I've begun writing an image classifier on my fork, if anyone wants to help.

**Comment by ethanbennett** [comment]
@jadenyjw I'm somewhat new to TFLearn (and machine learning in general), but I've been a writer for many years. Let me know if you'd like an editor for your work. I'd also be happy to "ghostwrite" tutorials, if you could provide me with some rough notes about each step you'd like included.
